linux/shim/init.go

```diff
@@ -347,3 +348,13 @@ func copyFile(to, from string) error {
 	_, err = io.Copy(tt, ff)
 	return err
 }
+
+func checkKillError(err error) error {
+	if err == nil {
+		return nil
+	}
+	if strings.Contains(err.Error(), "os: process already finished") {
```

Comment:
How is this expected as a return value from `unix.Kill`? This error message originates from the `os` package and is thrown from `(*Process).signal)`. That function returns this if the underlying syscall return `syscall.ESRCH`, should we just check for bother here?